packages/syft/src/syft/core/tensor/autograd/functions.py

```diff
@@ -2,31 +2,31 @@
 import numpy as np
 
 # relative
-from ..passthrough import inputs2child
 from ..util import implements
+from ..util import inputs2child
 from .tensor import AutogradTensor
 
 
 @implements(AutogradTensor, np.max)
 def npmax(*args: AutogradTensor, **kwargs: AutogradTensor) -> AutogradTensor:
-    args, kwargs = inputs2child(*args, **kwargs)
+    args, kwargs = inputs2child(*args, **kwargs)  # type: ignore
```

Comment:
These can be declared as *args: Tuple[Any, ...], **kwargs: Any to avoid having to use ignore.

Reply:
Fixed